bagua/torch_api/bucket.py

```diff
@@ -218,40 +215,27 @@ def append_decentralized_synchronous_op(
         when all the tensors within the bucket are marked ready.
 
         Args:
+            peer_weight (BaguaTensor): A tensor used for receiving and averaging model with peers, should be the same size
```

Comment:
we need better explanation on how this operator works, since it is not an inplace operation. The result will be in `peer_weight`. Users (who develop new algorithms) need to understand how this algorithm actually works and what they need to do after using this op